enjoy.py

```diff
@@ -43,9 +43,13 @@ def main():
                         help='Normalize reward if applicable (trained with VecNormalize)')
     parser.add_argument('--seed', help='Random generator seed', type=int, default=0)
     parser.add_argument('--reward-log', help='Where to log reward', default='', type=str)
-
+    parser.add_argument('--gym-packages', type=str, nargs='+', help='Additional external Gym environemnt package modules to import (e.g. gym_minigrid)')
```

Comment:
this should add a default value, otherwise CI fails

Reply:
Thanks for your response and sorry for my dirty hacks.  ;->
Let me provide a smaller PR on this without a wrapper fallback for dict obs first, if complicated.